core/go/internal/transportmgr/manager.go

```diff
@@ -256,6 +258,12 @@ func (tm *transportManager) LocalNodeName() string {
 // See docs in components package
 func (tm *transportManager) Send(ctx context.Context, send *components.FireAndForgetMessageSend) error {
 	ctx = log.WithComponent(ctx, "transportmanager")
+	return tm.SendWithNack(ctx, send, nil)
+}
+
+// See docs in components package
+func (tm *transportManager) SendWithNack(ctx context.Context, send *components.FireAndForgetMessageSend, errChan chan error) error {
```

Comment:
Comments on this interface above (around options and callback, rather than a slightly confused sync+channel interface with a close-channel as `success`)